examples/sugarscape_g1mt/sugarscape_g1mt/trader_agents.py

```diff
@@ -111,11 +113,7 @@ def is_occupied_by_other(self, pos):
             return False
         # get contents of each cell in neighborhood
         this_cell = self.model.grid.get_cell_list_contents(pos)
-        for a in this_cell:
-            # see if occupied by another agent
-            if isinstance(a, Trader):
-                return True
-        return False
+        return any(isinstance(a, Trader) for a in this_cell)
```

Comment:
@tpike3 check this out for simplification of the complexity tutorial.

Reply:
Works for me